### ISSUE TITLE
Update browser favicon to match current color theme

### ISSUE THREAD

**Issue by SlasherKG** [hoobs-org/HOOBS#1383]
**Is your feature request related to a problem? Please describe.**
Currently, the browser favicon remains orange even when choosing another color theme.

**Describe the solution you'd like**
It should update to match the current selection like the icon in the webpage itself does.

**Describe alternatives you've considered**
N/A

**Additional context**
![image](https://user-images.githubusercontent.com/11238501/125799434-963ce1f6-d651-4c5c-b3e4-b6a263b94630.png)

**Comment by mkellsy** [comment]
I tried really hard to do this. The issue is the color changing icon is SVG. The favorite icon is a raster image. Plus it is compiled into the base of the GUI app.

Also even if I was able to change the favicon, the browser caches this and it won't update as expected. Without clearing your browser history, it might not ever update.

I really hate marking issues as wont fix, but I have to on this one :(. If the technology ever changes to allow this, I will revisit.
